cmd/istioctl/inject.go

```diff
@@ -100,7 +100,8 @@ Example usage:
 
 			mesh, err := cmd.GetMeshConfig(client.GetKubernetesClient(), namespace, meshConfig)
 			if err != nil {
-				return err
+				return fmt.Errorf("could not find config map %s/%s. Is Istio system installed in %s?",
```

Comment:
maybe drop first namespace (that's not standard configmap name) and use quotes %q?